_posts/2022-09-10-executive-director-pyopensci-leah-wasser-updates.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: single
+title: "Big Changes for pyOpenSci - an update from the new executive director, leah wasser, what's next?"
+excerpt: "pyOpenSci has recently transitioned to a fiscal sponsor and has a new executive director - Leah Wasser. Learn more about our goals over the next few years and how to get involved with the community."
+author: "Leah Wasser"
+permalink: /blog/new-executive-director-leah-wasser
+header:
+    overlay_color: "#666"
+    overlay_filter: 0.6
+categories:
+  - updates
+  - blog-post
+---
+
+
+## pyOpenSci is now an independent community organization
+
+I am thrilled to announce that pyOpenSci is now a fiscally sponsored project of 
+Community Initiatives! This means that pyOpenSci now serves the community of 
+scientific python users and open source software maintainers. pyOpenSci is not 
+affiliated with any specific organization. I (Leah) will now be devoting ALL of my 
+(work :)) time to the pyOpenSci project full time as the official Executive Director. 
+
+## pyOpenSci’s current funding and structure
+
+I am grateful for 2 years of funding from the Sloan Foundation to 
+support this effort. Once I acquire additional funding, I 
+will be able to hire more staff to grow the organization. I am working with a 
+board of two right now: Tracy Teal (chair) and Karen Cranston. They both bring 
+tremendous experience building and supporting The Carpentries, among other 
+efforts; Tracy and Karen will guide me in propelling pyOpenSci forward. I also am 
+working with two incredible editors that have been crucial in supporting peer 
+review: David Nicholson and Ivan Ogasawara. In the upcoming months we will 
+recruit more editors to support our open source software review process. 
+Finally, I am also continuing to coordinate with and learn from the 
+well-established and successful rOpenSci. rOpenSci has set the precedent for 
+this type of organization in this space and I am grateful for their ongoing 
+support. 
+
+## pyOpenSci is not just peer review of open source software 
+Peer review is critical to the pyOpenSci mission but have bigger and broader 
+aspirations too. There are several organization 
+goals supported by the Sloan Foundation grant that we will be working on over 
+the next few years:
+* Create and grow a diverse and supportive community around a dynamic and 
+open peer review process and Python open science skills. Diversity is important and we plan to bake diversity focused programs into all of pyOpenSci activities from the beginning.
+* Streamline and improve the quality of smaller (think api wrappers, spatial data tools and domain specific tools rather than Numpy and Pandas) Python Scientific Open Source software tools that support scientific workflows.   
```

Comment:
Suggest changing this parenthetical to examples at the end of the sentence, so reader doesn't have to hold the first half of the sentence in their mind and so the examples illustrate your point:

* Streamline and improve the quality of smaller Python Scientific Open Source software tools that support scientific workflows: think api wrappers and domain specific tools rather than Numpy and Pandas

Reply:
oh got it.